lib/cirro_io/client/notifications_broadcast.rb

```diff
@@ -0,0 +1,8 @@
+module CirroIO
+  module Client
+    class NotificationsBroadcast < Base
+      has_one :channel
```

Comment:
you need to find a way that this channel maps to NotificationsChannel in json_api_client

On server side, we do it like this https://github.com/test-IO/cirro/blob/master/app/resources/api/v1/notifications_broadcast_resource.rb#L8